src/views/Settings.vue

```diff
@@ -270,9 +261,6 @@ export default defineComponent({
     },
     goToLaunchpad() {
       window.location.href = `${process.env.VUE_APP_LOGIN_URL}`
-    },
-    getDateTime(time: any) {
-      return DateTime.fromMillis(time).toLocaleString(DateTime.DATETIME_MED);
```

Comment:
No need to import DateTime now as not used.

Reply:
Done